spec/models/manageiq/providers/kubernetes/container_manager/container_group_spec.rb

```diff
@@ -34,6 +34,7 @@
     persistent_volume = FactoryGirl.create(
       :persistent_volume,
       :name   => "persistent_volume",
+      :parent_type => "ManageIQ::Providers::ContainerManager",
```

Comment:
That is correct since when we call `ems.persistent_volumes` we actually ignore the `parent_type`. 
I added another 2 persistent volumes with the correct parent type "ExtManagementSystem" to be tested.